### PR TITLE
GCP uninstall extension

### PR DESCRIPTION
**Pull request by prat98**
Added in instructions for uninstall extension and also moved install extension nodes section to install section as opposed to automation workloads section

### CODE REVIEW

File: stories/topics/proc-gcp-delete-extension-nodes.adoc [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+[IMPORTANT]
+====
+This action cannot be reversed.
+====
+
+Extension nodes are created as deployments in GCP Deployment Mananger. As such, the process to delete an Ansible Automation Platform extension node is the same as the process to delete the main Ansible Automation Platform deployment.  
+
+.Procedure
+. Select the main menu from the top left.
```
Comment (ianf77) [inline stories/topics/proc-gcp-delete-extension-nodes.adoc:13]: Select the main menu.

File: stories/topics/proc-gcp-delete-extension-nodes.adoc [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+. Select *Deployment Manager*.
+If you do not see *Deployment Manager*, select *View All Products*.
+. Tick the checkbox in front of the extension node you want to delete.
```
Comment (ianf77) [inline stories/topics/proc-gcp-delete-extension-nodes.adoc:16]: If the checkbox has a name it should be used here.

Comment (prat98) [reply stories/topics/proc-gcp-delete-extension-nodes.adoc:16]: Check box does not have a name unfortunately

**Comment by prat98** [comment]
Added in requested changes

**Review by zkayyali812 — APPROVED** [review]

**Review by ianf77 — APPROVED** [review]
LGTM

**Pull request merged**